static/js/alert_words.js

```diff
@@ -7,16 +7,25 @@ import * as people from "./people";
 let my_alert_words = [];
 
 export function set_words(words) {
-    my_alert_words = words;
+    my_alert_words = [];
+    for (const word of words) {
+        add_alert_word(word);
```

Comment:
Is there any problem with directly assigning words to `my_alert_words` as it was done before.

Reply:
currently it's like this - ["alert_word", "alert_word", "alert_word"]

but what we want actually in List widget is 

[{word: "alert_word"}, {word: "alert_word"}]

Reply:
so that we're able to sort them using the same function of `list_widget.js`

Reply:
I checked and `my_alert_words` is still simply a list only. `get_word_list` (which you have updated) returns it as `[{word: "alert_word"}, {word: "alert_word"}]`.

Reply:
yes, agreed.

Reply:
oh, i misunderstood your comment, sorry.

Reply:
Updated the PR accordingly.